zipline/protocol.py

```diff
@@ -57,7 +57,12 @@
     'sid',
 ]
 # Expected fields/index values for a dividend payment Series.
-DIVIDEND_PAYMENT_FIELDS = ['id', 'payment_sid', 'cash_amount', 'share_count']
+DIVIDEND_PAYMENT_FIELDS = [
+    'id',
+    'payment_sid',
+    'cash_amount',
+    'share_count'
```

Comment:
Probably want a trailing comma here.  I've been bitten many times by adding a list of strings like this, and then adding a new entry without adding another comma, which behaves unfortunately...

```
x = [
    'foo',
    'bar',
    'buzz' # Note the missing comma here.
    'bazz',
    'bozz',
]

In [5]: x
Out[5]: ['foo', 'bar', 'buzzbazz', 'bozz']
```